Snowflake.Data/Configuration/EasyLoggingConfigParser.cs

```diff
@@ -61,5 +66,22 @@ private void Validate(ClientConfig config)
                 EasyLoggingLogLevelExtensions.From(config.CommonProps.LogLevel);
             }
         }
+
+        private void CheckForUnknownFields(string fileContent)
+        {
+            // Parse the specified config file and get the key-value pairs from the "common" section
+            List<string> knownProperties = new List<string>();
+            foreach (var property in typeof(ClientConfigCommonProps).GetProperties())
```

Comment:
Done, refactored into a HashSet. Apparently Linq's ToHashSet is not available  for .NET Framework 4.7.1 so I was not able to use Linq for that

Reply:
So maybe let's use `ToList()` instead of `.ToHashSet();` and remove .NET Framework 471 handling. The code would be simpler and known properties have size of 2 elements so for know there aren't many benefits from the hash set.

Reply:
Sounds good, fixed